common/app/staticpages/StaticPages.scala

```diff
@@ -53,4 +53,21 @@ object StaticPages {
       ),
       groupedNewsletterResponses,
     )
+
+  def dcrSimpleNewsletterPage(
```

Comment:
Is there any background info on how this works? It's not something I'm particularly familiar with?

Reply:
As I understand it, `SimplePage` is a class which extends the `Page` and uses the `StandalonePage` trait and they are used to represent pages on the site which have metaData but no "content" (ie its not a `ContentPage` or a `GalleryPage` etc). They get constructed with a few fields and the class provide a set of helper methods return the meta data values based on the Configuration setting etc

I think the idea behind them is to provide an extendable container or helper class for keeping the meta data for all pages in a consistent format. For the purpose of this PR, the `dcrSimpleNewsletterPage` passed to the dcr service to build the rendering model.

The `StaticPages` object is a collection of these `SimplePages` - the existing `simpleNewslettersPage` is extended to include the `groupedNewsletterResponses` which the current frontend-rendered view needs as page for the page model. We don't need this for DCR, so it seemed to make sense to define the `dcrSimpleNewsletterPage` (same metadata but no `groupedNewsletterResponses` ) in`StaticPages` to avoid building it unnecessarily for a DCR request.